File: TypeScript/src/creational/singleton-and-monostate/monostate/module-b.ts
import { MyDatabase } from './db/implementation/my-database'
import { myDatabase as myDatabaseModuleA } from './module-a'

const myDatabase = new MyDatabase()

myDatabase.store({
  name: 'Julliete',
  email: 'jull.lopes@hotmail.com'
})
myDatabase.store({
  name: 'Gilmar',
  email: 'gilgil@hotmail.com'
})
myDatabase.store({
  name: 'Bianca',
  email: 'bianquinha@hotmail.com'
})

myDatabase.index()

console.log(myDatabaseModuleA === myDatabase)

File: Python/creational/singleton-and-monostate/monostate/module_b.py
from db.my_database import MyDatabase
from module_a import myDatabase as myDatabaseModuleA

if __name__ == '__main__':
    myDatabase = MyDatabase()

    myDatabase.store({
        'name': 'Julliete',
        'email': 'jull.lopes@hotmail.com'
    })
    myDatabase.store({
        'name': 'Gilmar',
        'email': 'gilgil@hotmail.com'
    })
    myDatabase.store({
        'name': 'Bianca',
        'email': 'bianquinha@hotmail.com'
    })

    # myDatabase.delete(1)
    myDatabaseModuleA.index()
    print()
    myDatabase.index()
    
    print(myDatabaseModuleA == myDatabase)
